CvGameCoreDLL/CvPlot.cpp

```diff
@@ -1067,23 +1067,44 @@ void CvPlot::updateSymbols()
 
 	if(maxYield>0)
 	{
-		int maxYieldStack = GC.getDefineINT("MAX_YIELD_STACK");
-		int layers = maxYield /maxYieldStack + 1;
-
 		CvSymbol *pSymbol= NULL;
-		for(int i=0;i<layers;i++)
+		
+		// merijn: 5x symbols
+		if (maxYield >= 5)
 		{
 			pSymbol = addSymbol();
 			for (int iYieldType = 0; iYieldType < NUM_YIELD_TYPES; iYieldType++)
 			{
-				int iYield = yieldAmounts[iYieldType] - (maxYieldStack * i);
-				LIMIT_RANGE(0,iYield, maxYieldStack);
-				if(yieldAmounts[iYieldType])
+				if (yieldAmounts[iYieldType])

```

Comment:
Can we make this an explicit boolean, i.e.
```
if (yieldAmounts[iYieldType] > 0
```

Reply:
No problem.

Reply:
Done